src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/LabelSelectorExpressions.java

```diff
@@ -0,0 +1,83 @@
+package com.cloudbees.jenkins.plugins.kubernetes_credentials_provider;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.fabric8.kubernetes.api.model.LabelSelector;
+import io.fabric8.kubernetes.api.model.LabelSelectorBuilder;
+
+/**
+ * Parser for <a href="https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors">Kubernetes Label Selectors</a>.
+ */
+class LabelSelectorExpressions {
+
+    /**
+     * Parse Kubernetes label selector expression.
+     * Example:
+     * <pre>
+     * app in (jenkins-dev, jenkins-all)
+     * </pre>
+     * @param selector label selector expression or null
+     * @return parsed label selector
+     */
+    @NonNull
```

Comment:
this should be unneeded [ref](https://github.com/jenkinsci/kubernetes-credentials-provider-plugin/blob/76d6f07759bf38a5b0d572b9e228feb32cbf32e9/src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/package-info.java#L28)